File: src/main/java/no/nav/syfo/api/intern/domain/RSOppfoelgingsdialog.java
package no.nav.syfo.api.intern.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class RSOppfoelgingsdialog {
    public Long id;
    public String uuid;

    public String sistEndretAvAktoerId;
    public LocalDateTime sistEndretDato;

    public String status;
    public RSVirksomhet virksomhet;

    public RSGodkjentPlan godkjentPlan = null;

    public RSPerson arbeidsgiver;
    public RSPerson arbeidstaker;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfoelgingsdialog/RSOppfoelgingsdialog.kt
package no.nav.syfo.api.v3.domain.oppfoelgingsdialog

import java.time.LocalDateTime

data class RSOppfoelgingsdialog(
    var id: Long? = null,
    var uuid: String? = null,

    var sistEndretAvAktoerId: String? = null,
    var sistEndretDato: LocalDateTime? = null,

    var status: String? = null,
    var virksomhet: RSVirksomhet? = null,

    var godkjentPlan: RSGodkjentPlan? = null,

    var arbeidsgiver: RSPerson? = null,
    var arbeidstaker: RSPerson? = null
)
